svthalia/concrexit
# Add Payments API v2

Part of #1428 

### Summary
Add Payments API v2. Currently in draft because I still have to write the test, but would be nice if the approach could already be reviewed.

### How to test
Steps to test the changes you made:
1. Go to the API docs
2. Try the endpoints
3. Check different authentication structures
